update_defconfig.sh

```diff
@@ -35,9 +35,20 @@ tama)
     DRM="true";;
 esac
 
+echo "================================================="
+echo "Your Environment:"
+echo "ANDROID_ROOT: ${ANDROID_ROOT}"
+echo "KERNEL_TOP  : ${KERNEL_TOP}"
+echo "KERNEL_CFG  : ${KERNEL_CFG}"
+echo "KERNEL_TMP  : ${KERNEL_TMP}"
 for device in $DEVICE; do \
-    rm -rf $KERNEL_TMP
-    mkdir -p $KERNEL_TMP
+    ret=$(rm -rf ${KERNEL_TMP} 2>&1);
+    ret=$(mkdir -p ${KERNEL_TMP} 2>&1);
+    if [ ! -d $KERNEL_TMP ] ; then
```

Comment:
Nit: please keep the braces here to maintain consistency.

Reply:
> Nit: please keep the braces here to maintain consistency.

Thank you, @pablomh !